### DISCUSSION TITLE
[4.0] Group specific redirection url after succesful login

### DISCUSSION THREAD

**Discussion by ElephantInTheRoomFirst** [joomla/joomla-cms#23326] [2. Cold Features]
### Is your feature request related to a problem? Please describe.
I need group specific url after successful login. There are good amount of issues related with it, including privacy, router etc.

The login form module provides a facility for an internal url. But it takes one URL for all groups. 

### Describe the solution you'd like
It will be good idea to extend it to for all groups. 

### Additional context
These are issues in similar context
https://www.collectiveray.com/joomla/joomla-tutorials/customizing-the-joomla-login-for-different-users.html
https://docs.joomla.org/How_do_you_redirect_users_after_a_successful_login%3F#Overriding
https://github.com/joomla/joomla-cms/issues/22215
https://github.com/joomla/joomla-cms/issues/22600

**Comment by tonypartridge** [comment]
Hmm, I needed the same so wrote a plugin.

However, we should be able to write in a repeatable subform for this if the community thinks it would be beneficial.

**Comment by ElephantInTheRoomFirst** [comment]
@tonypartridge Thanks for the reply.
Well in my case, I want to have a group specific dashboard. Then I checked on internet and found out that stock Joomla does support it partially. There are many people who are looking for it. 

Then I decided to check opinions of the community. My suggestion is to extend the current login form module to include group specific redirection menus.

**Comment by DavidBoggitt** [comment]
Doesn't this do what you want?

https://extensions.joomla.org/extensions/extension/access-a-security/site-access/redirect-on-login/

**Comment by ElephantInTheRoomFirst** [comment]
@DavidBoggitt Thanks for the reply.
I have checked the extension. It is unneccessary complicated and has many extra features. 
The stock joomla login form has login redirection menu, so it means joomla fellows are thinking in that direction. The feature can be extended to make it group specific. 
I saw enough people are searching for the feature. If the community finds it useful then we can add in the stock Joomla.

**Comment by ghost** [comment]
I guess the feature Request is for J4 so i changed the Title.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23326">issues.joomla.org/tracker/joomla-cms/23326</a>.</sub>

**Comment by ElephantInTheRoomFirst** [comment]
Thanks.
I have changed my design to ACL based dashboard rendering. Therefore the issue is no longer interesting for me. However it will be good to have the feature in J4. :)

**Comment by ghost** [comment]
> Therefore the issue is longer interesting

do you mean "no longer"?

**Comment by ghost** [comment]
> However it will be good to have the feature in J4. :)

At the End a Release Lead Decision.
